documentation/v0.2.0/components/introduction.md

```diff
@@ -12,9 +12,10 @@ GenAI Stack has two main components level abstraction:
 
 Check the components for detailed explaination on the components:
 
--   [ETL](https://genai-stack.aiplanet.com/components/data-extraction-and-loading)
--   [VectorDB](https://genai-stack.aiplanet.com/components/vector-database)
--   [Prompt Engine](https://genai-stack.aiplanet.com/components/prompt-engine)
--   [Retrieval](https://genai-stack.aiplanet.com/components/retrieval)
--   [Memory](https://genai-stack.aiplanet.com/components/memory)
--   [Model](https://genai-stack.aiplanet.com/components/llms)
+-   [ETL](etl)
```

Comment:
Shouldn't this be /components/etl ?

Reply:
introduction.md is already inside the components folder, even i dont know we have to check in gitbook only